department-of-veterans-affairs/vets-website
# Add trusts section

## Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Import Trusts - Section 8 into the Income & Asset Statement form (21P-0969)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86444

## Associated Pull Request(s)
- https://github.com/department-of-veterans-affairs/vets-json-schema/pull/948

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `income_and_assets_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/income_and_assets_form_enabled
4. Go to `http://localhost:3001/pension/apply-for-veteran-pension-form-21p-527ez/` in your browser

## How to verify
1. Go to chapter 8 - Trusts (/trusts-summary)
2. Verify that a trust can be added, edited, and deleted
3. Verify that the trusts are displayed on the 'Review' page

## What areas of the site does it impact?
Income & Asset Statement Application

## Screenshots
### Desktop
#### Summary Page
| No Trusts      | Trusts       |
| ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_ (19)](https://github.com/user-attachments/assets/bc04a3a2-d315-498c-93b2-26ed209bd52e)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (18)](https://github.com/user-attachments/assets/9e29fdb2-47fd-4b5a-a72e-a667097d3b0a)|


### Item Pages
| Trust Information | Trust Type | Trust Income |
| ----------- | ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_ (20)](https://github.com/user-attachments/assets/98e1b840-e72b-49ba-b17b-bc1cf29e6455)|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true](https://github.com/user-attachments/assets/e257ffbb-b2c0-45e3-8d6e-e0fd0b5c87fa)|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true (7)](https://github.com/user-attachments/assets/4ad99930-d760-44e2-b4fe-9159875b46a0)


| Trust Medical Expenses | Trust for Child | Trust Control |
| ----------- | ----------- | ----------- | 
|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true (8)](https://github.com/user-attachments/assets/7ed28165-b5f8-4663-8ca3-7a13f506d403)|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true (3)](https://github.com/user-attachments/assets/34654f19-8d55-4bc8-b81e-48ea4885bc97)|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true (4)](https://github.com/user-attachments/assets/3b8bc9a8-4708-491b-9bb3-bf6e3c652515)

| Has Added Funds | Added Funds (Conditional) |
| ----------- | ----------- | 
|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true (5)](https://github.com/user-attachments/assets/87728e0b-0207-45ef-bc56-77a120bebcd4)|![localhost_3001_income-and-asset-statement-form-21p-0969_trusts_0_trust-information_add=true (6)](https://github.com/user-attachments/assets/88783e16-9f33-48bd-818f-eac2d71f2fd9)


### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
